js/src/ui/Portal/portal.js

```diff
@@ -156,9 +153,13 @@ export default class Portal extends Component {
   handleClose = () => {
     const { hideClose, onClose } = this.props;
 
+    console.log('handleClose');
```

Comment:
Most likely this has stayed here by accident.

Would not like to make the console more noisy. Either use a solution like `import 'debug'` or remove it.